src/redux/services/schema.ts

```diff
@@ -44,6 +44,7 @@ export function getResourceSchema(resource: K8sResource, schemaVersion: string,
     k8sSchema = JSON.parse(
       readFileSync(path.join(String(userDataDir), path.sep, 'schemas', `${schemaVersion}.json`), 'utf-8')
     );
+    k8sSchemaCache = new Map<string, any | undefined>();
```

Comment:
k8SchemaCache.clear() is less drastic!? and you would have to clear the schemaCache also